kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkTask.java

```diff
@@ -217,7 +217,9 @@ public void put(Collection<SinkRecord> records) throws ConnectException {
 
       if (maybeSkipOnNullValue(record)) {
         if (reporter != null) {
-          reporter.report(record, new DataException("Cannot write null value record"));
+          String nullValueMsg = "Cannot write null value record";
+          reporter.report(record, new DataException(nullValueMsg));
+          log.warn("Errant record written to DLQ due to: {}", nullValueMsg);
```

Comment:
did we have a warning previously? This might generate too many messages if a user is skipping them (happens to have lot's of tombstones that you need to skip).

Reply:
We did not have a warning previously. There is also one in the TopicPartitionWriter for when a DataException is caught, but that one may not be as frequent so I think we may leave the other one.